client/tests/webdriver/specs/editLocation.spec.js

```diff
@@ -0,0 +1,48 @@
+import EditLocation from "../pages/location/editLocation.page"
+
+const EX_LOCATION_ID = "f2207d9b-204b-4cb5-874d-3fe6bc6f8acd"
+
+const EXAMPLE_LAT_LNG_TO_CONVERT = {
+  lat: "47.52963",
+  lng: "-52.94612"
+}
+
+const EXAMPLE_MGRS_TO_BE_EXPECTED = "22TCT5351665858"
+
+describe("When editing an existing location", () => {
+  it("Should see latitude and longitude label when format selected LAT_LON", () => {
+    EditLocation.open(EX_LOCATION_ID)
+    EditLocation.latLngLabel.waitForExist()
+    EditLocation.latLngLabel.waitForDisplayed()
+  })
+
+  it("Should correctly converts and displays both formats in the popover window", () => {
+    const latInput = EditLocation.latInputField
+    // known issue on chrome driver, setValue shouldn't append but it does (https://github.com/webdriverio/webdriverio/issues/3024)
+    latInput.setValue(
```

Comment:
Can't you do a clearValue first?

Reply:
Somehow missed that :+1:

Reply:
As it turns out it doesn't work either, see https://github.com/webdriverio/webdriverio/issues/4482#issuecomment-543332411